src/main/java/controllers/CommandLineInterface.java

```diff
@@ -324,13 +342,21 @@ public Schedule displayUserSchedules(List<Schedule> userSchedules) {
                     }
                     break;
                 case 'S':
-                    System.out.println("Saving this schedule in .ics format...");
-                    new ICSExporter().outputSchedule(currSchedule);
+                    System.out.println(
+                            "Please specify the name you'd like to save this Schedule under.");
+                    String icsFileName = scanner.next();
+                    System.out.println(
+                            "Saving this schedule in .ics format as " + icsFileName + ".ics ...");
```

Comment:
for some reason this outputs 'SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder"' in the cli. it still works through, so this might just be a quirk.

Reply:
yeah I think it's somethin with the ICS stuff. not really sure what the issue is, but works regardless